albumentations/augmentations/bbox_utils.py

```diff
@@ -119,29 +119,42 @@ def convert_bbox_to_albumentations(bbox, source_format, rows, cols, check_validi
 
     Args:
         bbox (list): bounding box
-        source_format (str): format of the bounding box. Should be 'coco' or 'pascal_voc'.
+        source_format (str): format of the bounding box. Should be 'coco', 'pascal_voc', or 'yolo'.
         check_validity (bool): check if all boxes are valid boxes
         rows (int): image height
         cols (int): image width
 
     Note:
         The `coco` format of a bounding box looks like `[x_min, y_min, width, height]`, e.g. [97, 12, 150, 200].
         The `pascal_voc` format of a bounding box looks like `[x_min, y_min, x_max, y_max]`, e.g. [97, 12, 247, 212].
+        The `yolo` format of a bounding box looks like `[x, y, height, width]`, e.g. [0.3, 0.1, 0.05, 0.07].
```

Comment:
Could you please add that `height` and `width` are normalized to one and `x`, `y` coordinates of the center of the box?

Reply:
`[x, y, height, width]` => `[x, y, width, height]`

Reply:
@ternaus Fixed, thanks for your comment